modules/programs/firefox.nix

```diff
@@ -743,6 +743,14 @@ in {
         message = "Container id must be smaller than 4294967294 (2^32 - 2)";
       })
 
+      {
+        assertion = length cfg.languagePacks != 0 && cfg.package != null;
```

Comment:
This is logically wrong, you probably want `cfg.languagePacks == [ ] || cfg.package != null` here.